modules/nw-operator-cr.adoc

```diff
@@ -23,239 +23,314 @@ endif::[]
 [id="nw-operator-cr_{context}"]
 = Cluster Network Operator configuration
 
-The configuration for the cluster network is specified as part of the Cluster Network Operator (CNO) configuration and stored in a custom resource (CR) object that is named `cluster`. The CR specifies the parameters for the `Network` API in the `operator.openshift.io` API group.
+The configuration for the cluster network is specified as part of the Cluster Network Operator (CNO) configuration and stored in a custom resource (CR) object that is named `cluster`. The CR specifies the fields for the `Network` API in the `operator.openshift.io` API group.
 
-ifdef::post-install-network-configuration[]
+The CNO configuration inherits the following fields during cluster installation from the `Network` API in the `Network.config.openshift.io` API group and these fields cannot be changed:
+
+- `clusterNetwork`: IP address pools from which pod IP addresses are allocated
+- `serviceNetwork`: IP address pool for services
+- `defaultNetwork.type`: Cluster network provider, such as OpenShift SDN or OVN-Kubernetes
+
+// For the post installation assembly, no further content is provided.
+ifdef::post-install-network-configuration,operator[]
 [NOTE]
 ====
-After cluster installation, you cannot modify the configuration for the cluster network provider.
+After cluster installation, you cannot modify the fields listed in the previous section.
 ====
-endif::post-install-network-configuration[]
+endif::[]
 ifndef::post-install-network-configuration[]
-You can specify the cluster network configuration for your {product-title} cluster by setting the parameter values for the `defaultNetwork` parameter in the CNO CR. The following CR displays the default configuration for the CNO and explains both the parameters you can configure and the valid parameter values:
+You can specify the cluster network provider configuration for your cluster by setting the fields for the `defaultNetwork` object in the CNO object named `cluster`.
+
+[id="nw-operator-cr-cno-object_{context}"]
+== Cluster Network Operator configuration object
+
+The fields for the Cluster Network Operator (CNO) are described in the following table:
+
+.Cluster Network Operator configuration object
+[cols=".^3,.^2,.^5a",options="header"]
```

Comment:
Looks like you might be able to shave a little colwidth off the first two columns if you wanted.